cabal.project

```diff
@@ -10,8 +10,8 @@ repository cardano-haskell-packages
     d4a35cd3121aa00d18544bb0ac01c3e1691d618f462c46129271bccf39f7e8ee
 
 -- The hackage index-state
-index-state: 2025-01-14T23:25:37Z
-index-state: cardano-haskell-packages 2025-01-13T22:30:16Z
+index-state: 2025-01-16T20:25:08Z
```

Comment:
Missing the flakes update

Reply:
Has been done on master, so rebase. 

Had lost track of this PR.

Reply:
How was it done on master if index state is updated in this PR?

Reply:
This PR no longer contains an index-state update. It just takes the index-state that master uses.